tests/job_applications/tests.py

```diff
@@ -1057,22 +1057,28 @@ def test_cancel(self):
         )
 
         cancellation_user = job_application.to_company.active_members.first()
-        email = job_application.email_cancel(cancelled_by=cancellation_user)
+        # mail.outbox = []  # Delete previous emails.
+        job_application.cancel(user=cancellation_user)
+        assert len(mail.outbox) == 2
+
         # To.
-        assert cancellation_user.email in email.to
-        assert job_application.sender.email in email.bcc
-        assert len(email.to) == 1
-        assert len(email.bcc) == 1
+        assert cancellation_user.email in mail.outbox[0].to
+        assert job_application.sender.email in mail.outbox[1].to
```

Comment:
Autant être précis et continuer de vérifier tous les destinataires.
```suggestion
        assert [cancellation_user.email] == mail.outbox[0].to
        assert [job_application.sender.email] == mail.outbox[1].to
```